src/pages/index.js

```diff
@@ -56,23 +60,175 @@ const cardSection = new Section(
   selectors.cardList
 );
 
-const createCard = (cardData) => {
-  const card = new Card(
-    {
-      data: cardData,
-      handleImageClick: () => {
-        popupWithImage.open(cardData);
-      },
-    },
-    selectors.cardTemplate
-  );
-  return card.generateCard();
-};
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "95813e96-588f-4415-8ffb-bcee955e5070",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getUserInfo()
+  .then((userData) => {
+    userInfo.setUserInfo({
+      name: userData.name,
+      about: userData.about,
+      avatar: userData.avatar,
+    });
+    userInfo.setUserInfo(userData.avatar);
+  })
+  .catch((err) => console.error(err));
+
+api
+  .getInitialCards()
+  .then((cards) => {
+    cardSection.renderItems(cards);
+  })
+  .catch((err) => {
+    console.error("Error fetching initial cards:", err);
+  });
 
 userInfoPopup.setEventListeners();
 newCardPopUp.setEventListeners();
 popupWithImage.setEventListeners();
-cardSection.renderItems(initialCards);
+confirmPopup.setEventListeners();
+profileImagePopup.setEventListeners();
+
+/* Functions */
+
+function createCard(cardData) {
+  const card = new Card(
+    cardData,
+    selectors.cardTemplate,
+    handleImageClick,
+    handleDeleteCard,
+    handleCardLike
+  );
+  const deleteCardbtn = document.querySelector(".card__delete-button");
+
+  if (deleteCardbtn) {
+    deleteCardbtn.addEventListener("click", () => confirmPopup.open());
+  }
+  return card.generateCard();
+}
+
+function handleImageClick(cardData) {
+  popupWithImage.open(cardData);
+}
+
+function handleDeleteCard(card) {
+  confirmPopup.setSubmitFunction(() => {
+    confirmPopup.setIsDeleting(true);
+    api
+      .deleteCard(card.getId())
+      .then(() => {
+        card._element.remove();
```

Comment:
`_element`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.